docs/ReleaseNotes.md

```diff
@@ -1,6 +1,28 @@
 # Acuminator Release Notes
 This document provides information about fixes, enhancements, and key features that are available in Acuminator.
 
+## Acuminator 2.3
+Acuminator 2.3 includes the bug fixes and enhancements described in this section, as well as the features that have been implemented in previous versions.
+
+### Enhancements
+In Acuminator 2.3, the following enhancements have been implemented:
+ - The [PX1036](diagnostics/PX1036.md) diagnostic has been added to check the correct naming of key field declarations in PK or FK API.
+ - The [PX1035](diagnostics/PX1035.md) diagnostic has been added to check and warn of multiple primary key declarations.
+ - The [PX1033] (diagnostics/PX1033.md) and [PX1034] (diagnostics/PX1034.md) diagnostics has been added to suggest adding PK or FK definition to a DAC.
+ - The [PX1016](diagnostics/PX1016.md) diagnostic has been added to check whether a graph extension has the IsActive method implemented.
+ - The Code Map has been improved as follows:
+   - The Code Map now indicates whether a graph is for mass processing operations and whether a graph node corresponds to a graph or a graph extension.
+   - The Code Map now indicates whether a DAC field is auto-incremented (has the PXDBIdentityt attribute).
+   - The following complementary information is now displayed in the Code Map about a graph view member: Whether the view is for mass processing operations, for setup form, or for filtering, whether the view is derived fom the base PXSelect or PXSelectReadOnly types.
```

Comment:
Imho it's better to say graph view instead of graph view member. I would also change the enumeration of indicators into something like a list:
Code Map now displays extra information about the graph views. You can see indicators for:
- Views for mass processing operations. // We frequently use term processing views
- Setup view //Views used to query for setup settings, I'm not sure if we need to mention setup forms here because these views are used by many data entry screens
- Filter views  //It's another established term, views derived from PXFilter
- Custom views derived from base PXSelect-types  // This are special custom views which were used some time ago to share common logic for different screens, for example, approval functionality. 
- Views derived from PXSelectReadOnly. // These views are not merged with graph cache, therefore they are marked as **Uncached** in the Code Map. This could be mentioned in the notes